qe_metrics/utils/product_utils.py

```diff
@@ -18,7 +21,10 @@ def products_from_file(products_file: str) -> List[Dict[Any, Any]]:
     products_dict = parse_config(path=products_file)
     products = []
     for name, queries in products_dict.items():
-        verify_queries(queries_dict=queries)
-        products.append({"product": ProductsEntity.get(name=name) or ProductsEntity(name=name), "queries": queries})
+        try:
+            verify_queries(queries_dict=queries)
+            products.append({"product": ProductsEntity.get(name=name) or ProductsEntity(name=name), "queries": queries})
+        except ValueError as err:
+            LOGGER.error(f"Error occurred parsing queries for product {name}: {err}")
```

Comment:
TODO:
since this is running application and we do not monitor the logs it is a good idea to have notification to slack on errors.

Reply:
@myakove Do you want me to add a TODO comment, or does this work? https://github.com/RedHatQE/qe-metrics/issues/71

I figure Slack will be used for more than just this error message, so an issue seems better suited.

Reply:
issue is enough; resolving